cinema/views.py

```diff
@@ -0,0 +1,42 @@
+from django.http import JsonResponse, HttpResponse
+from django.views.decorators.csrf import csrf_exempt
+from django.shortcuts import get_object_or_404
+from rest_framework.parsers import JSONParser
+from .models import Movie
+from .serializers import MovieSerializer
+
+@csrf_exempt
+def movie_list(request):
+    if request.method == 'GET':
+        movies = Movie.objects.all()
+        serializer = MovieSerializer(movies, many=True)
+        return JsonResponse(serializer.data, safe=False)
+
+    elif request.method == 'POST':
+        data = JSONParser().parse(request)
+        serializer = MovieSerializer(data=data)
+        if serializer.is_valid():
+            serializer.save()
+            return JsonResponse(serializer.data, status=201)
+        return JsonResponse(serializer.errors, status=400)
+
+
+@csrf_exempt
+def movie_detail(request, pk):
+    movie = get_object_or_404(Movie, pk=pk)
+
+    if request.method == 'GET':
+        serializer = MovieSerializer(movie)
+        return JsonResponse(serializer.data)
```

Comment:
Using `JsonResponse` here can be replaced with Django REST Framework's `Response` class for better integration and content negotiation.